src/qvimagecore.cpp

```diff
@@ -25,7 +25,7 @@ QVImageCore::QVImageCore(QObject *parent) : QObject(parent)
     sortDescending = false;
     allowMimeContentDetection = false;
 
-    randomSortSeed = 0;
+    baseRandomSortSeed = std::chrono::system_clock::now().time_since_epoch().count();
```

Comment:
As discussed, I believe the old approach is ok, and I believe it should rescramble every time the folder is re-opened.